### PR TITLE
update Kodein-Framework librairies

### PR DESCRIPTION
**Pull request by romainbsl**
- Adding Kodein-DB as a Kotlin multiplatform database
- Update Kodein-DI metadata path

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -304,10 +304,15 @@
     "category": "Math",
     "maven": "https://repo1.maven.org/maven2/com/ionspin/kotlin/bignum/"
   },
+  {
+    "github": "Kodein-Framework/Kodein-DB",
+    "category": "Storage",
+    "maven": "https://dl.bintray.com/kodein-framework/Kodein-DB/org/kodein/db/kodein-db/"
```
Comment (Alex009) [inline libraries.json:310]: it's already added into `DataBase` catagory.
if you change category in already exist item - it will be better

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: libraries.json (hunk 1 above)
Comment (romainbsl) [reply libraries.json]: fixed, but renamed `Database` instead.
Sorry, didn't checked as on https://libs.kmp.icerock.dev there wasn't Kodein-DB near to SQL libraires.
Plus, category are not sorted in the dropdown so didn't saw it.

File: libraries.json [hunk 2]
```diff
@@ -746,7 +746,7 @@
   },
   {
     "github": "Kodein-Framework/Kodein-DB",
-    "category": "DataBase",
+    "category": "Database",
```
Comment (Alex009) [inline libraries.json:749]: please change category to `Storage`

Comment (romainbsl) [reply libraries.json]: done

**Review by Alex009 — APPROVED** [review]

**Comment by Alex009** [comment]
thanks!

**Pull request merged**